lib/rubocop/cop/style/no_metaprogramming.rb

```diff
@@ -0,0 +1,73 @@
+# frozen_string_literal: true
+
+module RuboCop
+  module Cop
+    module Style
+      # Discourages the use of metaprogramming. It's designed
+      # to be enabled in select folders within your application. It was
+      # originally developed with the idea of "Business logic should minimize
+      # the amount of metaprogramming."
+      #
+      # This cop specifically will warn on usage of `.included`, `.inherited`,
```

Comment:
Looking at the actual code it seems to disable a few more things which should be mentioned here. The cop also needs some good/bad code examples.